### DISCUSSION TITLE
Section 3: make deploy not working on anvil

### DISCUSSION THREAD

**Discussion by TheBeanEffect** [Cyfrin/security-and-auditing-full-course-s23#73] [Q&A] [answered]
I'm using vs code, I have foundry, anvil launches properly, it doesn't show any errors. When I try to deploy using `make deploy` , I keep getting:
```
$   make deploy
bash: make: command not found
```
Has anyone had a similar problem and/or knows what to do? Any advice will be highly appreciated, I've been stuck since yesterday.

**Comment by johnsonstephan** [comment, marked as answer]
Hi @technicallybean, happy to help out here!

It appears the `make` command may not be installed or unavailable. 

First try to (re)install Make. Assuming you're using Homebrew you can run the following  `brew install make`.

If that doesn't work then it might be an issue with the PATH...

**Reply by TheBeanEffect** [reply]
Thank you for your suggestions!

If anyone else runs into this issue, I ended up following [this Git Bash guide](https://gist.github.com/evanwill/0207876c3243bbb6863e65ec5dc3f058) and that finally solved it.

**Reply by johnsonstephan** [reply]
No problem, I'm glad to hear that you solved it!

**Reply by clement-touzet** [reply]
For those on Windows (like me) there is this tutorial which I think is easier to follow : [https://linuxhint.com/install-use-make-windows/](https://linuxhint.com/install-use-make-windows/)

Basically, you have to :
- open a powershell as an admin
- install choco 
- install make via `choco install make`
- use `make deploy` in a git bash (or WSL) terminal
